tikv-client-common/src/security.rs

```diff
@@ -76,9 +77,12 @@ impl SecurityManager {
 
         let addr = SCHEME_REG.replace(addr, "");
 
+        // Set grpc.use_local_subchannel_pool will ensure every TiKV client instance can has it's own
+        // connection to TiKV server.
         let cb = ChannelBuilder::new(env)
             .keepalive_time(Duration::from_secs(10))
-            .keepalive_timeout(Duration::from_secs(3));
+            .keepalive_timeout(Duration::from_secs(3))
+            .raw_cfg_int(CString::new("grpc.use_local_subchannel_pool").unwrap(), 1);
```

Comment:
`raw_cfg_int` is not intended to be used directly according to its doc.I think it would be better to add a method grpc-rs to set it.

Reply:
OK, I will create a PR for grpc-rs first. But this is the easy way to set the configuration at rust-client side.